CHANGELOG.md

```
@@ -179,7 +218,8 @@ changes are included in and superseded by [0.4.0-alpha.1].
 - Made unavailable or failing platform probes report their state without
   stopping or changing the break timer.
 
-[Unreleased]: https://github.com/abhiksark/unfocus/compare/v0.4.0-alpha.1...dev
+[Unreleased]: https://github.com/abhiksark/unfocus/compare/v0.5.0-alpha.1...dev
```

markdownlint-cli2 (0.23.2)

[warning] Link and image reference definitions should be needed
Unused link or image reference definition: "unreleased"

(MD053, link-image-reference-definitions)